### ISSUE TITLE
Remove GIF Support

### ISSUE THREAD

**Issue by mazenchami** [infinitered/nsfwjs#818]
- [x] remove `@nsfw-filter/gif-frames`
- [x] remove `classifyGif`
- [x] update `README`
- [x] NSFWJS.com to exist without GIF support

**Comment by GantMan** [comment]
^ add update DOCs and NSFWJS.com to exist without this support.

**Comment by rosmcmahon** [comment]
why?

**Comment by GantMan** [comment]
Great question - the GIF library that was broken out was effective but unmaintained.  It also added a good bit of size and functionality that was specific to one technology.  If we support GIF why not support Video?  etc.  The rabbit hole was widening so we just clamped back down to the core of the service and left the door open for someone else to build ON TOP of this library to support a variety of media.

There's a LOT of options I can think of adding, but at last I decided against it - unless we get funding to support a wider array of features.
